modules/dashboard/test/TestPlan.md

```diff
@@ -2,3 +2,6 @@
    [Automate Test on Travis_CI]
 2. Check performance on the raisin bread dataset: make sure the dashboard page displays within 5 seconds or less.
    [Manual Test]
+3. Verify that if user has certain permission(s), the corresponding widget(s) will be displayed on the dashboard. [Automated Test]  
+4. Verify the links on the dashboard can navigate to the correct pages. [Automated Test]  
+5. Check the remaining content displayed on the dashboard.  
```

Comment:
What is "the remaining content"?

Reply:
just verified - this comment has been addressed in a commit.